app/Http/Controllers/UserController.php

```diff
@@ -91,11 +92,32 @@ public function updatePassword(Request $request)
         return redirect()->back()->with('message', __('general.successful_modification'));
     }
 
+    public function setCollegistType(Request $request)
```

Comment:
This seems too complicated. What are you trying to do here?

Reply:
This sets the collegist resident/extern status.
I simplified this with new functions in user model.

Reply:
however, still not works :/